code/script.js

```diff
@@ -313,13 +375,37 @@ const guess = (personToConfirm) => {
 // If you confirm, this function is invoked
 const checkMyGuess = (personToCheck) => {
   // 1. Check if the personToCheck is the same as the secret person's name
+    if (personToCheck === secret.name) {
   // 2. Set a Message to show in the win or lose section accordingly
+  winOrLoseText.innerHTML = `
+      <div class="winner-card"><img class="winner-img" src="images2/${secret.name}-big.jpg" />
+      <p class="win-or-lose-text">You goddamn right it's that dirty fish ${secret.name}!</p>
+      <p class="win-or-lose-text">Well played!</p></div>`
   // 3. Show the win or lose section
+      winOrLose.style.display = "flex"
```

Comment:
I'm curious to learn more what you did here? Why do you add "flex"? Perhaps a brief description in the code would have be helpful here.